CHANGELOG.txt

```diff
@@ -1,4 +1,6 @@
-1.2.1-SNAPSHOT
+2.0.1-SNAPSHOT
+
+2.0.0
```

Comment:
why `2.0`? `1.3` isn't more appropriate since there wasn't any breaking change?

Reply:
There is a breaking change. The `Codec` interface was changed: https://github.com/fauna/faunadb-jvm/commit/2b7a46e72294da672dfcfffd9289afc44c8f0981#diff-b25ef48a70d8ad0cbaf2152cea4cc634L51

Reply:
oh, completely forgot that, for things like that I created a tool to inspect c# assemblies, https://github.com/marrony/AssemblyDiff